### PR TITLE
Cloudslicemain 1

### PR DESCRIPTION
**Pull request by basimolimajeed**
removed unused inputs from the template. 
Do not merge yet, just review please @sebassem

### CODE REVIEW

File: azure_arc_servers_jumpstart/artifacts/Bootstrap.ps1 [hunk 1]
```diff
@@ -69,15 +69,15 @@ Start-Transcript -Path $Env:ArcBoxLogsDir\Bootstrap.log
 
 $ErrorActionPreference = 'SilentlyContinue'
 
-if ([bool]$vmAutologon) {
+#if ([bool]$vmAutologon) {
```
Comment (sebassem) [inline azure_arc_servers_jumpstart/artifacts/Bootstrap.ps1:72]: Why are we removing autologon ?

File: azure_arc_servers_jumpstart/bicep/main.bicep [hunk 2]
```diff
@@ -1,6 +1,18 @@
 @description('Azure AD tenant id for your service principal')
 //param spnTenantId string
-param spnTenantId string = tenant().tenantId
+//param spnTenantId string = tenant().tenantId
+
+var spnTenantId = tenant().tenantId
```
Comment (sebassem) [inline azure_arc_servers_jumpstart/bicep/main.bicep:5]: Let's keep it as a parameter and set the default value to `tenant().tenantId` to make the code reusable if we need to use it for a levelup

**Review by sebassem — APPROVED** [review]
LGTM, just minor comments:

- Lets remove the commented code if we don't need it anymore
- Why are we removing autologon ?
- Lets do the tenantId as a parameter to make the deployment reusable by any future levelups

File: azure_arc_servers_jumpstart/artifacts/Bootstrap.ps1 (hunk 1 above)
Comment (basimolimajeed) [reply azure_arc_servers_jumpstart/artifacts/Bootstrap.ps1:72]: I did not see the point of having autologon as in an input parameter. Why would we need it to be selected true/false. So I removed it to keep it always autologon. 
but I can put it back if there is a good reason

File: azure_arc_servers_jumpstart/bicep/main.bicep (hunk 2 above)
Comment (basimolimajeed) [reply azure_arc_servers_jumpstart/bicep/main.bicep:5]: ok

**Comment by basimolimajeed** [comment]
I will remove the commented code. What do you think about the vmautologon comment above?

**Review by sebassem — APPROVED** [review]

File: azure_arc_servers_jumpstart/artifacts/Bootstrap.ps1 (hunk 1 above)
Comment (basimolimajeed) [reply azure_arc_servers_jumpstart/artifacts/Bootstrap.ps1:69]: Kept autologin as default for now

File: azure_arc_servers_jumpstart/bicep/main.bicep (hunk 2 above)
Comment (basimolimajeed) [reply azure_arc_servers_jumpstart/bicep/main.bicep]: done

**Pull request merged**